TrkFilters/src/HelixFilter_module.cc

```diff
@@ -56,31 +56,37 @@ namespace mu2e
     double        _bz0;
     TrkFitFlag    _goodh; // helix fit flag
     std::string   _trigPath;
+    bool          _prescaleUsingD0Phi;
+    std::vector<float> _prescalerPar;
```

Comment:
A small struct would be better here - give the 3 parameters meaning names.   You can write fcl helper functions that allow you to read a table into a struct with a single function call in the calling code.  Let me know if you want to see an example.